src/content/docs/configuration/sched-ext.md

```diff
@@ -70,50 +74,147 @@ sudo systemctl start scx
 
-That's why having choices is one of the highlights from the sched-ext framework, so don't be scared to try the main ones and see which one works best for your use case, being ex: fps stability, maximum performance, responsiveness under intensive workloads etc.
+Being one of the heaviest schedulers yet released on sched-ext, it comes with a lot of features that add to his flexibility and capability. Tunability is one of them so you can adjust Rusty to your desires and use case.
+
+**Use cases:**
+
+- Gaming
+- Latency sensitive workloads
+- Desktop usage
+- Multimedia/Audio production
+- Latency sensitive workloads
+- Great interactivity under intensive workloads
+- Power saving
+
+For more information about what can be done with Rusty and his tunable flags. Check out the help page:
+
+```text
+scx_rusty --help
+```
+
+### [scx_lavd](<https://github.com/sched-ext/scx/tree/main/scheds/rust/scx_lavd>)
+
+**Developed by: Changwoo Min (multics69 [GitHub](<https://github.com/multics69>)).**
+
+**Brief introduction to LAVD from Changwoo:**
+
+***LAVD is a new scheduling algorithm which is still under development. It is
+motivated by gaming workloads, which are latency-critical and
+communication-heavy. It aims to minimize latency spikes while maintaining
+overall good throughput and fair use of CPU time among tasks.***
+
+**Use cases:**
+
+- Gaming
+- Audio Production
+- Latency sensitive workloads
+- Desktop usage
+- Great interactivity under intensive workloads
+- Power saving
+
+One of the main and awesome capabilities that LAVD includes is **Core Compaction.** which without going into technical details: When CPU usage < 50%, Currently active cores will run for longer and at a higher frequency. Meanwhile Idle Cores will stay in C-State (Sleep) for a much longer duration achieving less overall power usage.
+
+### [scx_bpfland](<https://github.com/sched-ext/scx/tree/main/scheds/rust/scx_bpfland>)
+
+**Developed by: Andrea Righi (arighi [GitHub](<https://github.com/arighi>))**
+
+A vruntime-based sched_ext scheduler that prioritizes interactive workloads. Highly flexible and easy to adapt, a deadline-based behavior can be achieved when lowlatency mode is enabled.
+
+Bpfland when making decisions on which cores to use, it takes in consideration their cache layout and which cores share the same L2/L3 cache leading to fewer cache misses = more performance.
+
+**Use cases:**
 
-### Which one do i choose?
+- Gaming
+- Latency sensitive workloads
+- Desktop usage
+- Multimedia/Audio production (Thanks to the low latency mode)
+- Great interactivity under intensive workloads
+- Power saving
 
-It depends but for mixed workloads meaning it could vary from gaming, programming, video editing, browsing etc. Rusty/Bpfland/Rustland or even LAVD.
+## Scheduler Profiles
 
-Gaming? then you'll have to choose what matters the most for you.
+:::note
+These profiles are only accessible by using [CachyOS Kernel Manager](/configuration/kernel-manager#sched-ext-gui):
```

Comment:
We should move this to kernel manager page instead, its more suitable as these presets are introduced from that program. 

The note is also quite vague, as while it is true these presets are a feature that was introduced from CachyOS Kernel Manager, it is also possible to replicate these presets by setting scx scheduler flags manually as this is what kernel manager does.